File: mobile/src/main/java/treehou/se/habit/ui/servers/sitemaps/list/SitemapSelectComponent.java
package treehou.se.habit.ui.servers.sitemaps.list;


import dagger.Subcomponent;
import treehou.se.habit.module.ActivityScope;
import treehou.se.habit.module.FragmentComponent;
import treehou.se.habit.module.FragmentComponentBuilder;
import treehou.se.habit.ui.servers.serverlist.ServersFragment;
import treehou.se.habit.ui.servers.serverlist.ServersModule;

@ActivityScope
@Subcomponent(
        modules = SitemapSelectModule.class
)
public interface SitemapSelectComponent extends FragmentComponent<SitemapSelectFragment> {

    @Subcomponent.Builder
    interface Builder extends FragmentComponentBuilder<SitemapSelectModule, SitemapSelectComponent> {
    }
}

File: mobile/src/main/java/treehou/se/habit/ui/servers/sitemaps/list/SitemapSelectComponent.kt
package treehou.se.habit.ui.servers.sitemaps.list


import dagger.Subcomponent
import treehou.se.habit.module.ActivityScope
import treehou.se.habit.module.FragmentComponent
import treehou.se.habit.module.FragmentComponentBuilder

@ActivityScope
@Subcomponent(modules = arrayOf(SitemapSelectModule::class))
interface SitemapSelectComponent : FragmentComponent<SitemapSelectFragment> {

    @Subcomponent.Builder
    interface Builder : FragmentComponentBuilder<SitemapSelectModule, SitemapSelectComponent>
}